src/featureFlags/flags/featureFlags.ts

```diff
@@ -9,4 +9,5 @@ export enum FeatureFlag {
   swapWidget = 'swap_widget_replacement_enabled',
   statsigDummy = 'web_dummy_gate_amplitude_id',
   detailsV2 = 'details_v2',
+  newContractForPWAT = 'new_contract_for_pwat',
```

Comment:
nit: get the latest UR contract version number and use that in the flag name such as `pwat_contract_vXX.XX`. This avoids when the next version comes out, us having to use the flag `new_new_contract_for_pwat`

Reply:
I actually think this is a better way to do it because with two prs we could update the contract without changing the feature flag. By changing both deprecated and nft router address to the current address we have now, then turn off feature flag, then update the new nft router to the new address. I think this beats making a new feature flag for every new contract upgrade - when we can just reuse this one

Reply:
resolved on slack - but will keep this naming convention since it has to match the current naming convention on statsig -however for future prs will change